engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -993,3 +993,246 @@ BEGIN
     DECLARE CONTINUE HANDLER FOR 1061 BEGIN END; SET @ddl = CONCAT('ALTER TABLE ', in_table_name); SET @ddl = CONCAT(@ddl, ' ', ' ADD KEY ') ; SET @ddl = CONCAT(@ddl, ' ', in_index_name); SET @ddl = CONCAT(@ddl, ' ', in_key_definition); PREPARE stmt FROM @ddl; EXECUTE stmt; DEALLOCATE PREPARE stmt; END;
 
 CALL `cloud`.`IDEMPOTENT_ADD_KEY`('i_user_ip_address_state','user_ip_address', '(state)');
+--
+-- Update Configuration Groups and Subgroups
+--
```

Comment:
none of the add columns below are idem potent

Reply:
Adding a column in an idempotent way is not supported our of the box :(

Reply:
there is, `IDEMPOTENT_ADD_COLUMN` , but soit.